pyoozie/oozie_api.py

```diff
@@ -0,0 +1,561 @@
+from __future__ import unicode_literals
+
+import logging
+import requests
+import simplejson as json
+
+from pyoozie.exceptions import OozieException
+from pyoozie.model import ArtifactType, Coordinator, CoordinatorAction, \
+    Workflow, WorkflowAction, parse_coordinator_id, parse_workflow_id
+from pyoozie.builder import _coordinator_submission_xml, _workflow_submission_xml
+
+logger = logging.getLogger('pyoozie.OozieAPI')
+logging.getLogger('requests').setLevel(logging.WARNING)
+
+
+class OozieAPI(object):
```

Comment:
I'll buy `OozieClient`, but I'm opposed to potentially ambiguous names like `API` or `Client` outside of very limited scopes (and even then, not a big fan).